solutions/banking/watson-scripts/assistant-project-creation.sh

```diff
@@ -7,7 +7,19 @@ WATSON_ASSISTANT_URL=$2
 
 token="$(echo "$IAM_TOKEN" | awk '{print $2}')"
 
-curl -X POST --location "$WATSON_ASSISTANT_URL/v2/assistants?version=2023-06-15" \
+ASSISTANT_ID=$(curl -X GET --location "$WATSON_ASSISTANT_URL/v2/assistants?version=2023-06-15" \
```

Comment:
maybe use some other curl flags here to make it more robust: `--retry 3 -fLsS` is what I tend to use on GETs - see https://curl.se/docs/manpage.html)

Reply:
This is a good idea - will incorporate it into my discovery scripts too.